src/components/balance/modals/ModalSelectAccount.vue

```diff
@@ -20,19 +20,19 @@
               tw-bg-transparent tw-placeholder-gray-300
               dark:tw-placeholder-darkGray-600
             "
-            :class="isH160 ? 'input-h160' : 'input-ss58'"
+            :class="isCheckMetaMask ? 'input-h160' : 'input-ss58'"
             type="text"
             spellcheck="false"
             :readonly="isReadOnly"
-            @focus="openOption = !isH160"
+            @focus="openOption = !isCheckMetaMask"
```

Comment:
Please don't hard code in the ethereum wallet name as MetaMask. Since we have plans to support other ethereum wallets like Trust, what about changing the variable name to `isEthWallet`?